src/main/java/core/basesyntax/FigureGenerator.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureGenerator {
+
+    private static final String[] colors = new String[]{"black", "white", "whatever"};
```

Comment:
we can use `enum` for colors - you can read about it later
But now - `Srting array` is enough :)

Reply:
Your wish is my command